Removed  MATS Data from this screen to avoid confusion

Jhernandez29 opened the pull request:
Ticket
https://github.com/US-EPA-CAMD/easey-ui/issues/6301

Changes
Updated src/components/QACertTestSummaryHeaderInfo/QAImportModalSelect/QAImportModalSelect.js react child Component. Remove key, value for mats.

Jhernandez29 committed:
Removed  MATS Data from this screen to avoid confusion and only be available on the QA Test

maxdiebold-erg commented on src/components/QACertTestSummaryHeaderInfo/QAImportModalSelect/QAImportModalSelect.js:
```diff
@@ -6,7 +6,6 @@ const QAImportModalSelect = ({ setImportTypeSelection }) => {
     { key: "select", name: "Select Data Type to Import" },
     { key: "file", name: "Import from File" },
     { key: "historical", name: "Import Historical Data" },
-    { key: "mats", name: "MATS Data" },
```
This removes the option from the _Cert Events, Extensions, & Exemptions_ screen **AND** the _Test Data_ screen. Per the acceptance criteria, it should only be removed from _Cert Events, Extensions, & Exemptions_. This component is used by the `QACertEventHeaderInfo` and `QACertTestSummaryHeaderInfo` components: a prop should be added that can be used to conditionally include this option, for example:
```javascript
    ...(qaType === "testSummary" ? { key: "mats", name: "MATS Data" } : {})
```

maxdiebold-erg replied:
Sorry, there was an error in my suggestion, but what you committed is correct when combined with the change in `DropdownSelection.js`.

maxdiebold-erg replied:
Just to note, what I _meant_ to put in my suggestion is:
```javascript
...(qaType === "testSummary" ? [{ key: "mats", name: "MATS Data" }] : [])
```
This would alleviate the need for the empty object check in `DropdownSelection.js` (because of how the spread operator works).

Jhernandez29 replied:
@maxdiebold-erg  If I do the ...(qaType === "testSummary" ? [{ key: "mats", name: "MATS Data" }] : []) code change there is still an empty option element. We don't want that. I think we should keep the if statement I added to skip any empty objects. 
![image](https://github.com/user-attachments/assets/1e7bd9cc-812d-43fd-9065-8f9d7ff84ca2)

Jhernandez29 replied:
Added ...(qaType === "testSummary" ? [{ key: "mats", name: "MATS Data" }] : []) change

maxdiebold-erg reviewed (changes requested):
Please see the comment below, this fix removes the option from both QA screens, when it should only be removed from one.

Jhernandez29 committed:
Updated DropdownSelection.js,
QACertEventHeaderInfo.js, QACertTestSummaryHeaderInfo.js, and QAImportModalSelect.js.

Jhernandez29 commented:
@maxdiebold-erg  
Revert the last commit and
Updated DropdownSelection.js,
QACertEventHeaderInfo.js, QACertTestSummaryHeaderInfo.js, and QAImportModalSelect.js.

maxdiebold-erg reviewed (dismissed):
Looks good, thanks for addressing that!

maxdiebold-erg committed:
Merge branch 'develop' into fix/#6301-qa-qce-tee

Jhernandez29 committed:
Updated QAImportModalSelect\QAImportModalSelect.js with spread operator. This way we don't need the empty check.

mark-hayward-erg committed:
Merge branch 'develop' into fix/#6301-qa-qce-tee

Jhernandez29 committed:
Merge branch 'develop' into fix/#6301-qa-qce-tee

maxdiebold-erg committed:
Merge branch 'develop' into fix/#6301-qa-qce-tee

Jhernandez29 committed:
Merge branch 'fix/#6301-qa-qce-tee' of https://github.com/US-EPA-CAMD/easey-ecmps-ui into fix/#6301-qa-qce-tee
